packages/react-native-codegen/src/parsers/flow/components/extends.js

```diff
@@ -50,17 +49,7 @@ function removeKnownExtends(
 // $FlowFixMe[unclear-type] there's no flowtype for ASTs
 type PropsAST = Object;
 
-function getExtendsProps(
-  typeDefinition: $ReadOnlyArray<PropsAST>,
-  types: TypeDeclarationMap,
-): $ReadOnlyArray<ExtendsPropsShape> {
-  return typeDefinition
-    .filter(prop => prop.type === 'ObjectTypeSpreadProperty')
-    .map(prop => extendsForProp(prop, types))
-    .filter(Boolean);
-}
-
 module.exports = {
-  getExtendsProps,
```

Comment:
if `extendsForProps` is used only by `getExtendsProps`, I'll move the helper to `props.js` as well.

Reply:
actually, it is used by `removeKnownExtends(...)` as well.. :)